src/utilities/__tests__/extendSchema-test.js

```diff
@@ -338,6 +338,36 @@ describe('extendSchema', () => {
     expect(printExtensionNodes(someScalar)).to.deep.equal(extensionSDL);
   });
 
+  it('extends scalars by adding specifiedBy directive', () => {
+    const schema = buildSchema(`
+      type Query {
+        foo: Foo
+      }
+
+      scalar Foo
+
+      directive @foo on SCALAR
+    `);
+    const extensionSDL = dedent`
+      extend scalar Foo @foo
+
+      extend scalar Foo @specifiedBy(url: "https://first.com/foo_spec")
+
+      extend scalar Foo @specifiedBy(url: "https://second.com/foo_spec")
```

Comment:
@IvanGoncharov Thank you for the link to [`UniqueDirectivesPerLocation.js`](https://github.com/graphql/graphql-js/blob/master/src/validation/rules/UniqueDirectivesPerLocation.js#L59-L60).  That is indeed working as expected, and verified by this new test case I just added: d0606579

Apologies for misremember during today's working group call, but the actual situation that I was concerned about is demonstrated here, where a scalar type is extended multiple times.

I just wanted to call this out and to get confirmation that this was valid, and the expected result, as well as ask for set of yes on this specific part of the [implementation](https://github.com/m14t/graphql-js/blob/specified-directive/src/utilities/extendSchema.js#L330-L346)

Thanks again.

Reply:
@IvanGoncharov I wanted to see if you had any thoughts on this.

Reply:
@m14t No worries. Thanks for adding the detail test.
Sorry for the delayed reply and thanks for pinging 👍 
I'm traveling through Asia ATM so can disappear from time to time please feel free to ping me if I don't respond in a reasonable time

I'm trying to finish a few last features for `15.0.0` release and then I will review this PR in detail. Since it's not a breaking change I want to include it in `15.1.0` (if `15.0.0` don't result in any major problems we can release `15.1.0` in 1 or 2 weeks).

Reply:
Sounds good. Holding on this till `15.1.0` makes sense to me.

 I'll keep an eye out for `15.0.0` and if I don't hear from you within a week after that, I'll ping again.  Enjoy your travels!

Reply:
FWIW: this has been rebased off of `v15.0.0-rc.2`